awsenergylabelercli/awsenergylabelercli.py

```diff
@@ -106,17 +112,18 @@ def get_arguments():
                                                dest='single_account_id',
                                                action='store',
                                                type=aws_account_id,
+                                               default=os.environ.get('SINGLE_ACCOUNT_ID'),
                                                help='Run the labeler on a single account. '
                                                     'Mutually exclusive with --landing-zone-name argument.')
     parser.add_argument('--region',
                         '-r',
-                        default=None,
+                        default=os.environ.get('REGION'),
                         type=security_hub_region,
                         required=False,
                         help='The home AWS region, default is None')
     parser.add_argument('--frameworks',
                         '-f',
-                        default=["aws-foundational-security-best-practices"],
+                        default=os.environ.get('FRAMEWORKS', ["aws-foundational-security-best-practices"]),
```

Comment:
If the default argument here is a list then we should cast the env variable in a list. Env vars are always only string so some transformation would be required here.